### DISCUSSION TITLE
ACEScg and EXR

### DISCUSSION THREAD

**Discussion by attila3d** [ImageMagick/ImageMagick#2913] [Help]
I have a frame rendered in Houdini with ACES enabled as exr.  https://drive.google.com/file/d/1ylr6BXsuufKqpfKVfz1YbgIiLF4Gkeq-/view?usp=sharing

After processing it  (with any operation or just passing the image through, while keeping it exr) with Image Magick the colorspace seems to be different from the original.

https://drive.google.com/file/d/1tWl7jIPRvktb4ZOUfx-0wHt0CWcKoRdb/view?usp=sharing

Is there support for exr images stored in ACEScg? If I run identify -verbose on either the source or the processed image I get Colorspace: RGB, however based on my understanding it should be ACEScg, which is still linear but not RGB.
It also seems there is no option for -colorspace ACEScg

Is there any workaround for this? 
Thanks

**Comment by snibgo** [comment]
For colour management, IM uses ICC profiles, managed by an LCMS library. ICC profiles are available for ACES colorspaces, but your file does not seem to contain any ICC profile. Exiftool and IM "-verbose info:" seem to find no metadata that says your image is encoded in ACEScg.
